pkg/api/types_swagger_doc.go

```diff
@@ -0,0 +1,956 @@
+		"imagePullSecrets":              "ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this PodSpec. If specified, these secrets will be passed to individual puller implementations for them to use.  For example, in the case of docker, only DockerConfig type secrets are honored.",
+	}
+}
+
+func (PodStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                  "PodStatus represents information about the status of a pod. Status may trail the actual state of a system.",
+		"message":           "A human readable message indicating details about why the pod is in this state.",
+		"reason":            "A brief CamelCase message indicating details about why the pod is in this state. e.g. 'OutOfDisk'",
+		"startTime":         "Date and time at which the object was acknowledged by the Kubelet. This is before the Kubelet pulled the container image(s) for the pod.",
+		"containerStatuses": "The list has one entry per container in the manifest. Each entry is currently the output of `docker inspect`. This output format is *not* final and should not be relied upon. when we have done this.",
+	}
+}
+
+func (PodStatusResult) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":       "PodStatusResult is a wrapper for PodStatus returned by kubelet that can be encode/decoded",
+		"status": "Status represents the current information about a pod. This data may not be up to date.",
+	}
+}
+
+func (PodTemplate) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodTemplate describes a template for creating copies of a predefined pod.",
+		"template": "Template defines the pods that will be created from this pod template",
+	}
+}
+
+func (PodTemplateList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "PodTemplateList is a list of PodTemplates.",
+	}
+}
+
+func (PodTemplateSpec) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodTemplateSpec describes the data a pod should have when created from a template",
+		"metadata": "Metadata of the pods created from this template.",
+		"spec":     "Spec defines the behavior of a pod.",
+	}
+}
+
+func (Probe) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "Probe describes a liveness probe to be examined to the container.",
+		"initialDelaySeconds": "Length of time before health checking is activated.  In seconds.",
+		"timeoutSeconds":      "Length of time before health checking times out.  In seconds.",
+	}
+}
+
+func (RBDVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod",
+		"monitors":  "Required: CephMonitors is a collection of Ceph monitors",
```

Comment:
The json tag name diffs from the struct field name, so the "CephMonitors" doesn't make sense here